tests/lib/__init__.py

```diff
@@ -553,10 +553,10 @@ def main():
             entry_points=dict(console_scripts=['{name}={name}:main'])
         )
     """.format(name=name)))
-    return _vcs_add(script, version_pkg_path, vcs)
+    return _vcs_add(script, version_pkg_path, vcs, p4port)
 
 
-def _vcs_add(script, version_pkg_path, vcs='git'):
+def _vcs_add(script, version_pkg_path, vcs='git', p4port=None):
```

Comment:
Is there a way to structure this function so you don't need to add vcs-specific arguments?